auto_report/__main__.py

```diff
@@ -66,7 +68,7 @@ def cli():
         logger=logger
     )
     releases = release_stats.get(
-        from_date="now-6w/d",
+        from_date=f"now-{TIME_RANGE_IN_WEEKS}w/d",
```

Comment:
We can get this from env to make it more flexible, but this is still an improvement to the current version nevertheless :)